csi/moac/replica.ts

```diff
@@ -136,10 +136,16 @@ export class Replica {
   async destroy() {
     log.debug(`Destroying replica "${this}" ...`);
     if (!this.pool) {
-      throw new Error('Cannot offline a replica that has not been bound');
+      throw new Error('Cannot destroy a replica that has not been bound');
+    }
+    if (!this.pool.node.isSynced()) {
+      // We don't want to block the volume life-cycle in case that the node
+      // is down - it may never come back online.
+      log.warn(`Faking the destroy of "${this}" because it is unreachable`);
```

Comment:
My only worry here is what happens if the old nexus is not really offline, just having some network issues, and if for some reason it tries to write to the replicas, eg: update the "golden number". 
eg, a rebuild process could be ongoing and at the end it does reconfigure and bumps up the golden number while they are are already being used by another "newer" nexus on a different node.
A way to solve this could be to have the replicas tell the nexus via the data plane, "you're out".
A temporary way forward for now, could be checking if there is no rebuild ongoing? Can we assume that the nexus would not issue any other IO to the children since we've been called to unpublish?